### ISSUE TITLE
Layout/IndentAssignment false positive when assigning multiple values

### ISSUE THREAD

**Issue by Eneroth3** [rubocop-hq/rubocop#6911]
When assigning multiple values on one line to a value defined on the below line, as shown here, the Layout/IndentAssignment cop has a false positive.

```ruby
a = b =
  c
```

The reason why I am making such assignments in the first place is due to the history of the SketchUp Ruby API. When creating a command (a toolbar button) we previously defined a path to a small icon and a path to a large icon, as there are two toolbar scales users can chose from. For the last couple of years though vector icons have been supported, and the large and small icon path typically has the same value, but still needs to be both assigned.

```ruby
save_cmd.large_icon = save_cmd.small_icon =
  File.join(PLUGIN_DIR, "images", "tb_save#{LUI.icon_file_extension}")
```
--------

## Expected behavior

Code passes with no offense as the subsequent line is indented.

## Actual behavior

Layout/IndentAssignment offense detected, due to right-hand-side not being indented.

## Steps to reproduce the problem

Check following code with RuboCop:
```ruby
save_cmd.large_icon = save_cmd.small_icon =
  File.join(PLUGIN_DIR, "images", "tb_save#{LUI.icon_file_extension}")
```

## RuboCop version

```
rubocop -V
0.63.1 (using Parser 2.5.1.2, running on ruby 2.4.2 x64-mingw32)
```
(Latest version compatible with [rubocop-sketchup](https://github.com/SketchUp/rubocop-sketchup) it seems)

**Comment by baelter** [comment]
Confirmed

**Comment by buehmann** [comment]
@baelter Can you give us an example? I can't reproduce this with the example above and RuboCop v0.75.0 (seems to be fixed since https://github.com/rubocop-hq/rubocop/pull/7371).

**Comment by baelter** [comment]
My mistake
